src/common/components/controls/insights-command-button.scss

```diff
@@ -21,3 +22,21 @@ button.insights-command-button {
         }
     }
 }
+
+// high contrast overrides to make a.insights-command-button
+// match button.insights-command-button
+a.insights-command-button {
```

Comment:
From what I can tell the selector at line 6 and this selector at line 28 are one-in-the-same. What do you think about combining them? (For example, moving the media query and its contents before the closing bracket at line 24.)

Reply:
I had it that way to begin with and separated them for clarity but am happy to move it back!